README.md

```diff
@@ -7,3 +7,178 @@ Status](https://github.com/maltezfaria/DataFlowTasks.jl/workflows/CI/badge.svg)]
+                mul!(view(A,rj,rk), adjoint(view(A,ri,rj)), view(A,ri,rk))
+            end
+        end
+    end
+
+    # Construct the factorized object
+    return Cholesky(A,'U',zero(LinearAlgebra.BlasInt))
+end
+```
+
+When it will come to actually parallelize the code, we would only have with DataFlowTasks to wrap function calls within a `@dspawn`, and add a synchronization point at the end. The parallelized code will be :
+
+```julia
+using DataFlowTasks
+using LinearAlgebra
+function cholesky_dft!(A, ts)
+    m,n = size(A)
+    n%ts != 0 && error("Tilesize doesn't fit the matrix")
+    tn = round(Int, n/ts)
+
+    for ti in 1:tn
+        ri = tilerange(ti)
+
+        # Diagonal cholesky serial factorization (I)
+        @dspawn cholesky!(@RW(view(A,ri,ri))) label="chol ($ti,$ti)"
+
+        # Left blocks update (II)
+        L = adjoint(UpperTriangular(view(A,ri,ri)))
+        for tj in ti+1:tn
+            rj = tilerange(tj)
+            @dspawn ldiv!(@R(L), @RW(view(A,ri,rj))) label="ldiv ($ti,$tj)"
+        end
+
+        # Submatrix update (III)
+        for tj in ti+1:tn
+            for tk in tj:tn
+                rj = tilerange(tj)  ;  rk = tilerange(tk)
+                @dspawn mul!(@RW(view(A,rj,rk)), @R(adjoint(view(A,ri,rj))), @R(view(A,ri,rk))) label="mul ($tj,$tk)"
+            end
+        end
+    end
+    DataFlowTasks.sync()
+    # Construct the factorized object
+    return Cholesky(A,'U',zero(LinearAlgebra.BlasInt))
+end
+```
+
+The code below shows how to use this `cholesky_dft!` function, how to profile the program and get the most information from the visualization. 
+
+```julia
+import DataFlowTask as DFT
+
+# DataFlowTasks environnement setup
+DFT.reset!()
+DFT.enable_log()
+DFT.setscheduler!(DFT.JuliaScheduler(50))
+
+# Context
+n  = 4096
+ts = 512
+A = rand(n, n)
+A = (A + adjoint(A))/2
+A = A + n*I
+
+# Compilation
+cholesky_dft!(copy(A), ts)
+
+# Reset environnement
+DFT.resetlogger!()
+GC.gc()
+
+# Real work to be analysed
+cholesky_dft!(A ,ts)
+
+# Plot
+DFT.plot(categories=["chol", "ldiv", "mul"])
+```
+
+## Profiling
+
+We can illustrate the parallelization implied by those modifications. `DataFlowTasks.jl` comes with 2 main profiling tools whose outputs for the case presented above, with a matrix of size (2000, 2000) divided in blocks of (500, 500), are as follows :
+
+![Trace Plot](example.png)
+![Dag Plot](exampledag.svg)
+
+We'll cover in details the usage and possibilities of the visualization in the documentation.
+
+Note that the visualization tools are not loaded by default, it requires a `Makie` backend and/or `GraphViz` loaded in the REPL. It's meant to be used in development, so it won't pollute the environment you want to use DFT in.
+
+# Performances
+
+We compare the performances achieved with this version of the Cholesky factorization with the MKL one, and we obtain the next figure. Here the blocks are of size (256, 256).
+
+![Perf](scalability_lfaria.png)
```

Comment:
Has this image has been obtained with the simplified code above? If it comes from the complete high-performance implementation in `TiledFactorization`, we should say so here.